lib/broadside/target.rb

```diff
@@ -16,27 +16,16 @@ class Target
       :predeploy_commands,
       :scale,
       :service_config,
-      :tag,
```

Comment:
whoops; i should have waited for your answer before merging, but i'll put it back in a separate PR